Live/LiveEventWithDVR/Program.cs

```diff
@@ -3,47 +3,42 @@
 
 using Azure;
 using Azure.Identity;
```

Comment:
Looks like you have removed Event grid, but there is a lot of stuff in the Readme about Event Grid support still.